### PR TITLE
Fix: jenkinsfile cannot detected win unittest failure status

### PR DESCRIPTION
**Pull request by YifeiZuo01**
### Description
Jenkinsfile cannot detect windows bash command return code not 0 and failure the build as expected due to BAT command's property. This PR is trying to fix the problem.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -228,6 +228,7 @@ def winTest(String pythonVersion) {
                 pip freeze
                 python setup.py install
                 coverage run -m unittest discover -s test -vvv
+                if %errorlevel% neq 0 exit /b %errorlevel%
```
Comment (tommikas) [inline Jenkinsfile]: Have you tested this structure works?

Comment (YifeiZuo01) [reply Jenkinsfile]: I will test it in another PR and remove those changes later in that PR

Comment (YifeiZuo01) [reply Jenkinsfile]: it woks.
```
10:07:31 [windows]   File "C:\Jenkins\ws\mbed-flasher_PR-70
10:07:31 [windows] 
10:07:31 [windows] FAILED (errors=33, skipped=8)
[Pipeline] [windows] echo
10:07:31 [windows] Caught exception: hudson.AbortException: script returned exit code 1
[Pipeline] [windows] step
10:07:32 [windows] [Set GitHub commit status (universal)] FAILURE on repos [GHRepository@30fd12cd[description=mbed device flasher application,homepage=<null>,name=mbed-flasher,license=<null>,fork=false,size=701,milestones={},language=Python,commits={}
```

**Comment by tommikas** [comment]
The only additional suggestion is that you should consider if there are other lines in that also need that check to propagate the failure out correctly.

**Comment by YifeiZuo01** [comment]
@tommikas  basically each line in that windows bat should be checked, 
1. if activate the virtualenv failed, I guess it would report, but not totally sure
2. after successfully activate, if sth during installation failed, unittest would fail as well, the side effect for only added after unittest, is making it harder to debug.
3. normally it would be too much to add that fix line everywhere,

**Comment by tommikas** [comment]
> normally it would be too much to add that fix line everywhere,

If a failure on that line would cause the run to fail and the failure wouldn't get detected, that sounds like a reason to add the check. It's annoying and ugly, but if it's necessary...

Anyway, up to you.

**Pull request merged**